src/app/shell/admin-tools/data/admins/admins.component.ts

```diff
@@ -98,6 +104,44 @@ export class AdminsComponent implements OnInit, OnDestroy {
       relativeTo: this.route,
       queryParams: { role: AdminRoleUkrReverse[event.tab.textLabel] }
     });
+    this.setDisplayedColumns();
+  }
+
+  private setTabOptions(): void {
+    const queryRole = this.route.snapshot.queryParamMap.get('role');
+    this.adminParams.tabTitle = queryRole ? AdminRoleUkr[queryRole] : AdminRoleUkr.ministryAdmin;
+
+    switch (queryRole) {
+      case undefined:
+      case AdminRole.ministryAdmin:
+        this.tabIndex = 0;
+        break;
+      case AdminRole.regionAdmin:
+        this.tabIndex = 1;
+        break;
+      case AdminRole.territorialCommunityAdmin:
+        this.tabIndex = 2;
+        break;
+    }
+  }
+
+  private setDisplayedColumns(): void {
+    const isActionsInList = this.displayedColumns.includes('actions');
+
+    if (this.role === Role.techAdmin && isActionsInList) {
+      return;
+    }
+
+    if (this.role === Role.ministryAdmin) {
+      if (this.adminParams.tabTitle === AdminRoleUkr.ministryAdmin && isActionsInList) {
+        this.displayedColumns = this.displayedColumns.filter((value: string) => value !== 'actions');
+        return;
+      } else if (isActionsInList) {
+        return;
+      }
+    }
+
+    this.displayedColumns.push('actions');
```

Comment:
Am I correct?

tech admin always has actions, and the rest of admins does not ()for roles that currently exist)?